packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -256,20 +256,22 @@ export const visualizeSharedTree: VisualizeSharedObject = async (
 	sharedObject: ISharedObject,
 	visualizeChildData: VisualizeChildData,
 ): Promise<FluidObjectNode> => {
-	const sharedTree = sharedObject as ISharedTree;
-	const contentSnapshot = sharedTree.contentSnapshot();
+	const sharedTree = sharedObject as ITreeInternal;
 
-	// Root node of the SharedTree's treeview. Assume there is only one root node.
-	const treeView = contentSnapshot.tree[0];
+	// Root node of the SharedTree's content.
+	const treeView = sharedTree.exportVerbose();
+	// TODO: this visualizer doesn't consider the root as a field, and thus does not display the allowed types or handle when it is empty.
```

Comment:
Filed a task for this, if you'd like to reference it: https://dev.azure.com/fluidframework/internal/_workitems/edit/26472

Reply:
Done.